core-model/src/main/java/com/sequenceiq/cloudbreak/domain/InstanceStatus.java

```diff
@@ -1,5 +1,5 @@
 package com.sequenceiq.cloudbreak.domain;
 
 public enum InstanceStatus {
-    DECOMMISSIONED, REGISTERED, UNREGISTERED, TERMINATED
+    DECOMMISSIONED, REGISTERED, UNREGISTERED, TERMINATED, STOPPED_BY_HAND
```

Comment:
please rename this status to UNKNOWN - we cannot be sure if it really was stopped manually